eventuals/task.h

```diff
@@ -238,34 +252,54 @@ struct _TaskFromToWith {
     K_ k_;
     std::tuple<Args_...> args_;
 
-    Callback<
-        Action,
-        std::optional<std::exception_ptr>&&,
-        Args_&&...,
-        std::optional<
-            std::conditional_t<
-                std::is_void_v<From_>,
-                std::monostate,
-                From_>>&&,
-        std::unique_ptr<void, Callback<void*>>&,
-        Interrupt&,
+    // The 'dispatch_' might be both function that creates an eventual
+    // or argument that should be passed throw 'Task' in the 'Start' method
+    // of 'Task::Continuation' used in 'Task::Success'.
```

Comment:
```suggestion
    // The 'dispatch_' is a `std::variant` because its either a function that 
    // creates an eventual or the value from 'Task::Success' that should be 
    // passed on to the continuation.
```